pkg/util/prompt.go

```diff
@@ -18,9 +18,14 @@ func SetInputScanner(scanner *bufio.Scanner) {
 func GetInput(defaultValue string) string {
 	inputScanner.Scan()
 	input := inputScanner.Text()
+	value := strings.TrimSpace(input)
+
+	// Check for surrounding single or double quotes and remove them.
+	if len(value) >= 2 && (value[0] == '\'' && value[len(value)-1] == '\'' || value[0] == '"' && value[len(value)-1] == '"') {
```

Comment:
Isn't this much easier with strings.Trim()? 

value := strings.Trim(s, `"' `)

Reply:
Can you say a little more about what you're trying to prevent? An extra quote on the front or back of a filepath?

Reply:
It's not about filepath, `util.GetInput` is used in many places.

If user gives this input `"'test"`, then DDEV should read `'test`, just like it's done with bash, not `test` which would be the result of `strings.Trim`.

Reply:
Oh, sorry, that makes sense. But then how is it legitimate to remove single and double quotes anyway, if we assume that the user provided them?

Reply:
Should we maybe be using a filepath-specific input function for the import-db and import-files?